src/gui/widgets/GroupBox.cpp

```diff
@@ -99,35 +99,18 @@ void GroupBox::updatePixmap()
 	QColor bg_color = QApplication::palette().color( QPalette::Active,
```

Comment:
Could you please make this background color themeable too? `p.background();` should do the trick again.